### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by mertcihankurel**
### Tic Tac Toe Has Started!

1-) Created all linter folders
2-) Added all required working folders

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+p 'Hello World!'
```
Comment (kitpao) [inline bin/main.rb]: - [x] Please follow the [indicated guide in Pathwright](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) To make your file executable
image

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  Enabled: false
+Style/HashTransformKeys:
+  Enabled: false
+Style/HashTransformValues:
+  Enabled: false
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
+Metrics/PerceivedComplexity:
+  Enabled: false
```
Comment (kitpao) [inline .rubocop.yml]: - [x] Remember you should not change the files for the linters, please use the original files in [Microverse Repo](https://github.com/microverseinc/linters-config/tree/master/ruby) as stated in Pathwright
image

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @mcihadkurel and @Haywhizzz 

You did well following the new Git Flow 👏 

You are missing some details in your setup, please check them below 🔽 

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Comment by krys2fa** [comment]
## Good job so far @mcihadkurel and @Haywhizzz  👏 
Your project has a descriptive PR and good commit messages as well. 💪 

### Status: Changes Required ♻️ 
There's just one more change to make to get this approved. 

- [REQUIRED ] The executable file should work. That is it should display a message when `bin/main.rb` is run in the terminal ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone1/execute_bin.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).

That's all. Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by Haywhizzz** [comment]
We have made the main.rb executable

**Review by voscarmv — APPROVED** [review]
# Status: Project Approved ✨ 😎👍 💯 ✔️

- Excellent work applying the requested changes!

### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**